src/content/pages/explore.mdx

```diff
@@ -6,136 +6,129 @@ subtitle: At EuroPython, many attendees will come with their families. While not
-- Mostly safe, with no no-go stations or anything similar
+- More information can be found [here](https://ep2024.europython.eu/where#getting-around-in-prague).
+- You can use your regular card to travel around by train within Prague.
+- Public transport is generally fast and reliable.
+- After midnight, the metro stops running, but buses are available.
+- The public transport system is mostly safe, with no areas considered off-limits.
 
 ### Do I need to rent a car?
 
 Some people may consider renting a car, but:
 
-- In Prague, this is unnecessary. It’s hard to find parking, and it’s expensive
-- You can rent a car at the airport or other places in the city (like you normally do in other cities)
-- If you want to visit other towns, you can use trains as well (they’re mostly cheap)
+- In Prague, it's generally unnecessary due to limited parking availability and high costs.
+- Cars can be rented at the airport or other places in the city, similar to most other cities.
+- If you want to visit other towns, trains are a convenient and economical option.
 
 ### Be prepared for the weather
 
-If you want to explore the city, you will be exposed to the weather. So here are some answers to frequent questions like: What’s the weather like? What should you carry with you?
+If you want to explore the city, you should be prepared for the weather. Here are answers to some common questions:
 
-- Usually 30°C+, so carry water
-- There are no water fountains or public toilets, so carry more water
-- Use sunscreen
-- It rarely rains, but when it does, it’s shower rain. It’s best to just hide somewhere until it stops (it usually stops after half an hour or so).
+- Temperatures are usually above 30°C, so remember to carry water.
+- Public water fountains (and toilets) are rare, so remember to carry more water.
+- Use sunscreen.
+- It rarely rains, but when it does, it tends to be brief showers. Find shelter until it passes, usually within about half an hour.
 
 ## What to do while in Prague?
 
-There are plenty of things you can do in Prague. Depending on your preferences, you can find things that interest you and your group.
+There are plenty of things to enjoy in Prague. Depending on your preferences, you can find things that interest you and your group.
 
-### For culture lovers
+### For culture and art lovers
 
-- Summer cinema
-- Theater festival Za dveřmi 2024
-
-### For art lovers
-
-- DOX
-- Soví mlýny
-- National Museum
-- Jazz Dock for live jazz on a boat
-- Žižkov area for bars and pubs
-- House of the Black Madonna: cafe shop in a building that’s built in cubist style (there was a cubist architecture movement in Prague and it’s quite famous and unique)
+- Summer cinema (Check some available options [here](https://www.praguest.com/en/outdoor-cinemas-in-prague))
+- Theater festival [Za dveřmi 2024](https://zadvermi.cz/)
+- [DOX](https://www.dox.cz/en)
+- [Museum Kampa](https://www.museumkampa.cz/en/)
```

Comment:
@clytaemnestra I've changed the Soví mlýny to Museum Kampa. Since the museums occupy the buildings that were formerly the Sova’s Mills, I thought people would more likely be looking for what's going on in the museum. Not sure if it makes sense.

Reply:
sure, SGTM